torch/_meta_registrations.py

```diff
@@ -190,6 +192,103 @@ def meta_bernoulli(self, *, generator=None, out):
     return out
 
 
+@torch.library.impl(meta_lib, "convolution_overrideable")
```

Comment:
I think this should be `"convolution"` instead, no? @albanD

`convolution_overrideable` is intended to be removed, and `convolution` should be implemented directly for any new backends.

Reply:
I think we should yes.
RIght now, it gets the generic implementation for Meta that ends up calling into `convolution_overrideable` and this one throws an error (I guess that's why this is the one that @xiurobert implemented).
We should be able to override the Meta implementation for convolution directly indeed.

Reply:
I updated this in 30080b52912ded7e846d1e9994aa9a46069112e7, tests are passing locally